README.md

```diff
@@ -16,23 +16,10 @@ Top-level
 - crates/xmtp-keystore - first crate, implements the Keystore API in Rust
 - crates/xmtp-proto - Generated code for handling XMTP protocol buffers
 - crates/xmtp-networking - API client for XMTP's GRPC API, using code from `crates/xmtp-proto`
-- bindings/wasm - depends on libxmtp to generate a WASM library and bindings
 - bindings/xmtp_rust_swift - Swift bindings
 
 ## Rust Keystore QuickStart
 
 - cd `crates/xmtp-keystore`
 - `cargo test`
 
-## WASM QuickStart
```

Comment:
Let's add in a new section about `xmtpv3` - can literally just point to the README there after short intro.